### DISCUSSION TITLE
Are FileRoutes lazy loaded?

### DISCUSSION THREAD

**Discussion by andreevnikola** [solidjs/solid-start#817] [Q&A] [answered]
Hi! I am curios to know are FileRoutes lazy loaded? This is important for the performance and i really like the idea of file routing but if the routes are not lazy loaded this is not the right choice for my use case.

**Comment by ryansolid** [comment, marked as answer]
Yes atleast in the browser. We do different things on the server. Keep in mind route data is not so it can fetch in parallel.

**Comment by Feel-ix-343** [comment]
Is there a way to preload routes? I want my app to feel snappy AF
